app/assets/javascripts/beak/widgets/ractives/button.coffee

```diff
@@ -297,9 +297,9 @@ RactiveButton = RactiveWidget.extend({
 
     standardButton:
       """
-      <button id="{{id}}" type="button" style="{{dims}}"
+      <button id="{{id}}" type="button" style="{{dims}}" {{attrs}}
               class="netlogo-widget netlogo-button netlogo-command{{#widget.oldSize}} old-size{{/}}{{# !isEnabled }} netlogo-disabled{{/}} {{errorClass}} {{classes}}"
-              on-click="@this.fire('activate-button')">
+              on-click="@this.fire('activate-button')" aria-disabled="{{# !isEnabled }}true{{else}}false{{/}}">
```

Comment:
Isn't this just `aria-disabled="{{isEnabled}}"`?  This patterned gets repeated a lot below.